lib/interface.rb

```diff
@@ -0,0 +1,88 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'adapter'
+
+# Understands how to interact with the user
```

Comment:
great that you separated this functionality into a separate class, but again this UI is way more than was asked for.